### PR TITLE
feat: Add gcp/kubernetes configs

### PR DESCRIPTION
**Pull request by dcramer**
Please forgive us world, we have no idea what we're doing here.

### CODE REVIEW

File: kubernetes/zeus/kube-lego/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kube-lego
+  namespace: kube-lego
+data:
+  # modify this to specify your address
+  lego.email: "ops@sentry.io"
```
Comment (mattrobenolt) [inline kubernetes/zeus/kube-lego/configmap.yaml]: I assume not possible to actually make this configurable? Would be nice for this to not blast our mailing list if someone else decides to run this themselves without editing it.

File: kubernetes/zeus/zeus/nfs-pv.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  name: nfs
+spec:
+  capacity:
+    storage: 500Gi
+  accessModes:
+    - ReadWriteMany
+  storageClassName: standard
+  nfs:
+    path: /
+    server: 10.43.247.96
```
Comment (mattrobenolt) [inline kubernetes/zeus/zeus/nfs-pv.yaml]: Not sure if this is brittle or not. Just pointing it out. It's also publishing an internal IP address for the NFS host if there was some RCE. Probably not practically a concern.

File: kubernetes/zeus/kube-lego/configmap.yaml (hunk 1 above)
Comment (dcramer) [reply kubernetes/zeus/kube-lego/configmap.yaml]: i think the way this becomes configurable would be through templates and Helm-- outside of that you'd set this via the CLI when you create the recipe but thats a pita.

File: kubernetes/zeus/zeus/nfs-pv.yaml (hunk 2 above)
Comment (dcramer) [reply kubernetes/zeus/zeus/nfs-pv.yaml]: I couldn't find a way to make this easily discoverable unfortunately.

**Pull request merged**